include/libcyphal/transport/session/svc_sessions.hpp

```diff
@@ -0,0 +1,99 @@
+/// @file
+/// Defines the Service Session interfaces for Transport Layer implementations.
+///
+/// @copyright
+/// Copyright (C) OpenCyphal Development Team  <opencyphal.org>
+/// Copyright Amazon.com Inc. or its affiliates.
+/// SPDX-License-Identifier: MIT
+
+#ifndef LIBCYPHAL_TRANSPORT_SESSION_SVC_SESSION_HPP_INCLUDED
+#define LIBCYPHAL_TRANSPORT_SESSION_SVC_SESSION_HPP_INCLUDED
+
+#include "session.hpp"
+
+namespace libcyphal
+{
+namespace transport
+{
+inline namespace session
+{
+
+/// @brief Declares service request RX session parameters.
+///
```

Comment:
Again, I think it is better to write no doc comment at all than to write a comment that adds no information.

Reply:
I need double check what will happen if there are docs for members on an entity but without its own docs...